src/diffusers/pipelines/stable_diffusion/__init__.py

```diff
@@ -1,5 +1,30 @@
 # flake8: noqa
-from ...utils import is_transformers_available
+from dataclasses import dataclass
+from typing import List, Union
+
+import numpy as np
+
+import PIL
+from PIL import Image
+
+from ...utils import ModelOutput, is_transformers_available
+
+
+@dataclass
+class StableDiffusionOutput(ModelOutput):
```

Comment:
I'd probably use `StableDiffusionPipelineOutput` for clarity, even if it's more verbose.

```suggestion
class StableDiffusionPipelineOutput(ModelOutput):
```

Reply:
+1